targets/UnrealMarketplacePlugin/templates/PlayFabCpp/core/PlayFab_API.cpp.ejs

```diff
@@ -98,12 +95,11 @@ void UPlayFab<%- api.name %>API::On<%- apiCall.name %>Result(FHttpRequestPtr Htt
     FPlayFabCppError errorResult;
     if (PlayFabRequestHandler::DecodeRequest(HttpRequest, HttpResponse, bSucceeded, outResult, errorResult))
     {
-<%- getResultActions("        ", apiCall)
+<%- getResultActions("        ", apiCall, false)
 %>        SuccessDelegate.ExecuteIfBound(outResult);
     }
     else
     {
         ErrorDelegate.ExecuteIfBound(errorResult);
     }
-}
-<% } %>
+}<% } %>
```

Comment:
I think I actually prefer to have a final empty line at the end of the file.
It provides a very trivial thing where, if I add anything to the end of this file later, it won't display that I changed the last code-line of the file.
Also, perhaps more relevant, I'm generally against changing lines that don't need to be changed, so I'd prefer if you put this one back the way it was.
Thanks.